data/language/dutch.txt

```diff
@@ -66,7 +66,7 @@ STR_0064    :Attractie
 STR_0065    :Omgekeerde monorail
 STR_0066    :Attractie
 STR_0067    :Attractie
-STR_0068    :Heartline-twisterachtbaan
+STR_0068    :Heartline-Twisterachtbaan
```

Comment:
Was goed. 'twisterachtbaan' is een zelfstandig naamwoord en hoort met kleine letter geschreven te worden.